app/views/skateparks/show.html.erb

```diff
@@ -175,8 +168,8 @@
 
 
 function initMap() {
-  console.log(<%= @lat_long[0] %>)
-  var myLatLng = {lat: <%= @lat_long[0] %>, lng: <%= @lat_long[1] %>}
+  var latLong = <%= @lat_long %>
```

Comment:
We can change `@lat_long` to `@skatepark.get_lat_long`. What about renaming the method to just `Skatepark#lat_long`?